examples/imitation_learning_openbot/utils.py

```diff
@@ -135,26 +136,26 @@ def plot_tracking_performance_spatial(poses_current, poses_desired, plot_path):
     plt.savefig(plot_path, bbox_extra_artists=[legend], bbox_inches="tight")
 
 
-def plot_tracking_performance_temporal(poses_current, poses_desired, plot_path):
+def plot_tracking_performance_temporal(poses_tracked, poses_desired, plot_path):
```

Comment:
`poses_actual`

Reply:
Updated the names to location instead of poses as we are only passing in locations.